lib/mv_opentelemetry/plug.ex

```diff
@@ -106,7 +107,11 @@ defmodule MvOpentelemetry.Plug do
   @spec handle_stop_event(_ :: any(), _ :: any(), %{conn: Plug.Conn.t()}, Access.t()) :: :ok
   def handle_stop_event(_, _, %{conn: conn} = meta, opts) do
     ctx = OpentelemetryTelemetry.set_current_telemetry_span(opts[:tracer_id], meta)
-    Span.set_attribute(ctx, "http.status", conn.status)
+
+    Span.set_attributes(ctx, %{
+      "http.status" => conn.status,
+      Trace.http_status_code() => conn.status
```

Comment:
This is the biggest change, I made a mistake with "http.status" instead of http.status_code, so for a while both fields will be present.